.github/workflows/test.yml

```diff
@@ -98,6 +98,9 @@ jobs:
             - name: Print Firefox version
               run: firefox --version
 
+            - name: Authenticate with npm
```

Comment:
Why do we need to authenticate with npm?

Reply:
Tachometer benchmark is pointing out stale npm authentication